addons/mail/static/src/discuss/discuss.js

```diff
@@ -141,6 +144,11 @@ export class Discuss extends Component {
                 : this.MODES.MEMBER_LIST;
     }
 
+    async onFileUploaded(file) {
+        await this.threadService.updateAvatar(this.thread.id, file.data);
+        this.notification.add(_t("The avatar has been updated!"), { type: "success" });
```

Comment:
Not a big deal, IMO I prefer more brief and to-the-point wording, e.g. `Avatar changed!`.

Reply:
Haha, I would suggest keeping the words since the `Avatar changed!` could be too informal in this case.